.github/workflows/release.yml

```diff
@@ -214,15 +214,15 @@ jobs:
 
       - name: Compress Artifact
         run: |
-          tar -czf $(pwd)/resolc-wasm32-unknown-emscripten.tar.gz -C ./target/wasm32-unknown-emscripten/release/ \
-            resolc.js \
-            resolc.wasm \
-            resolc_web.js
+          mkdir -p resolc-wasm32-unknown-emscripten
+          mv ./target/wasm32-unknown-emscripten/release/resolc.js ./resolc-wasm32-unknown-emscripten/
+          mv ./target/wasm32-unknown-emscripten/release/resolc.wasm ./resolc-wasm32-unknown-emscripten/
+          mv ./target/wasm32-unknown-emscripten/release/resolc_web.js ./resolc-wasm32-unknown-emscripten/
 
       - uses: actions/upload-artifact@v4
         with:
           name: resolc-wasm32-unknown-emscripten
```

Comment:
Uses the same name for all three artifacts, is this fine?

Reply:
The commands move the files to the `resolc-wasm32-unknown-emscripten` folder, they will have different names.